### ISSUE TITLE
fix(a11y/stepper): colour contrasts not accessible

### ISSUE THREAD

**Issue by Marine-Fabien** [adeo/mozaic-design-system#1268]
### I am opening an issue for

Styles (CSS, SCSS, ...)

### Package version

@mozaic-ds/styles@1.48.1

### Description

The colours contrasts of stepper are not fully accessible. 

```
.mc-stepper__item--current .mc-stepper__indicator
```

<img width="1182" alt="Capture d’écran 2022-11-22 à 17 10 47" src="https://user-images.githubusercontent.com/77010633/203368007-2f5d6056-118a-499f-b515-ff13335096e0.png">

```
.mc-stepper__title
```

<img width="1182" alt="Capture d’écran 2022-11-22 à 17 10 09" src="https://user-images.githubusercontent.com/77010633/203368002-b1429d17-31da-4b7f-8027-e0713b8158ab.png">

### GitHub repository

https://github.com/adeo/mozaic-design-system

### To Reproduce

Check the colour contrasts with a RGAA tool for test it.

### Is this a regression?

- [ ] Yes
- [X] No
- [ ] I don't know

**Comment by caqueste** [comment]
Duplicate issue
